### DISCUSSION TITLE
Implementation of CSV of domains for "CORS_ORIGIN" environment variable

### DISCUSSION THREAD

**Discussion by ksoltanidev** [directus/directus#15371] [Feature Requests]
Hi there ! 

As the documentation says, we are supposed to be able to use a CSV of domains for the "CORS_ORIGIN" environment variable. But this feature was not implemented.
![image](https://user-images.githubusercontent.com/62473886/188179880-c0e53d5b-90c7-4db8-a391-29a534a0aaa2.png)

I opened a Pull Request #15370 to update it. But I've been said it might be open to discussion. So... here I am. 🙂

I followed the suggested pattern here : https://www.npmjs.com/package/cors#configuring-cors-w-dynamic-origin
The function check if the Origin is in the CORS_ORIGIN array and tries to map it.
If there is no origin, I guess we let the request through ? 

## Type of Change

- [x] Bugfix
- [ ] Improvement
- [x] New Feature
- [ ] Refactor / codestyle updates
- [ ] Other, please describe:

## Requirements Checklist

- [ ] New / updated tests are included
- [x] All tests are passing locally
- [x] Performed a self-review of the submitted code

If adding a new feature:

- [ ] Documentation was added/updated

**Comment by rijkvanzanten** [comment]
> But this feature was not implemented.

We didn't implement anything specific, because express' cors module already does that natively 🙂 

See "Array" under `origin`: https://www.npmjs.com/package/cors#configuration-options

**Reply by ksoltanidev** [reply]
After further tests. It is indeed working as expected. 
The environment variable was not properly set with a "array:" prefix. 
A small link here (or a tip) to the [environment-syntax-prefix](https://docs.directus.io/self-hosted/config-options.html#environment-syntax-prefix) would be nice.
![image](https://user-images.githubusercontent.com/62473886/188406228-6b651cde-155a-46e9-b58f-907fc4f8ffb5.png)

**Reply by benhaynes** [reply]
CC @erondpowell
